mir_eval/transcription.py

```diff
@@ -138,6 +138,173 @@ def validate(ref_intervals, ref_pitches, est_intervals, est_pitches):
                          "value")
 
 
+def validate_intervals(ref_intervals, est_intervals):
+    """Checks that the input annotations to a metric look like time intervals,
+    and throws helpful errors if not.
+
```

Comment:
"errors" -> "warnings"

Is this function redundant with [util.validate_intervals](https://github.com/craffel/mir_eval/blob/master/mir_eval/util.py#L636)?